deal stage and postgres updates

fivetran-joemarkiewicz opened the pull request:
This PR includes the following updates:

- Addition of the `deal_stage` model in response to [dbt_hubspot Issue](https://github.com/fivetran/dbt_hubspot/issues/30)
- Postgres compatibility
- Addition of the `hubspot_deal_company` variable as the dataset being used to test this addition did not contain this table and felt it was appropriate to include the option to disable if desired.

fivetran-jamie reviewed (approved):
looks good to me! with this and the service PR, should we increase the version to 0.3.1?

fivetran-joemarkiewicz commented:
Updated to v0.3.1 as well as dag and yml cleanup for ticket models (switched var to ref so dag isn't as *spaghetti* like. Finally, updated docs to reflect updates.

fivetran-jamie reviewed (approved):
lgtm
